### ISSUE TITLE
Todos

### ISSUE THREAD

**Issue by ariG23498** [ml-gde/jflux#14]
- [x] Offloading models https://github.com/google/flax/discussions/4273#discussioncomment-10892910
- [ ] TPU sharding
- [ ] T5 and CLIP Flax code -- logits checking (#15)
